### PR TITLE
Pin botocore version

### PR DESCRIPTION
**Pull request by ledor473**
Fixes for #5175 

I wasn't sure if I had to change it in `setup.py` as well... Mostly because the recent changes for Kombu were only in `requirements.txt` and that `setup.py` has a different boto3 version

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,5 +1,6 @@
 bleach==2.1.2
 boto3==1.4.7
+botocore<1.8.0,>=1.7.0
```
Comment (john-bodley) [inline requirements.txt]: You should pin to a specific version here.

**Comment by ledor473** [comment]
Running `pip3 freeze` gave me the version `1.7.48`. I've updated the PR with that version

**Comment by mistercrunch** [comment]
If you know a certain version is not working with Superset for sure, we should add a range in `setup.py` as well. So say if there are known issues with `1.8.0` I would add <1.8.0 in `setup.py` as well.

**Comment by ledor473** [comment]
@mistercrunch should I put the same boto3 version in `setup.py`? Just so there's no discrepancy between the two?

**Comment by mistercrunch** [comment]
`requirements.txt` should be pinned as much as possible, `setup.py` should be unpinned, but define version ranges when we know that >={some version} isn't compatible for sure.

**Comment by ledor473** [comment]
Sounds good! I've updated the PR

**Review by john-bodley — APPROVED** [review]

**Comment by jeffreythewang** [comment]
It seems like `boto3` verisions `1.4.6` and `1.4.8` are both incompatible with `botocore>=1.7.0, <1.8.0`, and only `1.4.7` satisfies the requirement. Does it make sense to pin `boto3` to `1.4.7`?

**Comment by mistercrunch** [comment]
@jeffreythewang if this is the only version that work then yes for sure

Now I'm thinking that ideally Superset would not have any dependencies on boto, only `extra_requires`, but let's just get this working first. I think I hit this issue last week but had no time to investigate...

**Pull request merged**